libraries/core_libs/network/rpc/Test.h

```diff
@@ -19,33 +19,13 @@ class Test : public TestFace {
   explicit Test(std::shared_ptr<taraxa::FullNode> const& _full_node) : full_node_(_full_node) {}
   virtual RPCModules implementedModules() const override { return RPCModules{RPCModule{"test", "1.0"}}; }
 
-  virtual Json::Value insert_dag_block(const Json::Value& param1) override;
-  virtual Json::Value get_dag_block(const Json::Value& param1) override;
   virtual Json::Value get_sortition_change(const Json::Value& param1) override;
-  virtual Json::Value get_nf_blocks(const Json::Value& param1) override;
   virtual Json::Value send_coin_transaction(const Json::Value& param1) override;
-  virtual Json::Value get_num_proposed_blocks() override;
   virtual Json::Value get_account_address() override;
-  virtual Json::Value get_account_balance(const Json::Value& param1) override;
   virtual Json::Value get_peer_count() override;
   virtual Json::Value get_node_status() override;
   virtual Json::Value get_packets_stats() override;
-  virtual Json::Value get_node_version() override;
-  virtual Json::Value get_node_count() override;
-  virtual Json::Value get_all_peers() override;
   virtual Json::Value get_all_nodes() override;
-  virtual Json::Value get_vote_weight(const Json::Value& param1) override;
-  virtual Json::Value place_vote(const Json::Value& param1) override;
-  virtual Json::Value get_votes(const Json::Value& param1) override;
-  virtual Json::Value draw_graph(const Json::Value& param1) override;
-  virtual Json::Value get_transaction_count(const Json::Value& param1) override;
-  virtual Json::Value get_executed_trx_count(const Json::Value& param1) override;
-  virtual Json::Value get_executed_blk_count(const Json::Value& param1) override;
-  virtual Json::Value get_dag_size(const Json::Value& param1) override;
-  virtual Json::Value get_dag_blk_count(const Json::Value& param1) override;
-  virtual Json::Value get_pbft_chain_size() override;
-  virtual Json::Value get_pbft_chain_blocks(const Json::Value& param1) override;
-  virtual Json::Value get_db_stats() override;
```

Comment:
so based on what did you remove rpc calls ?

Reply:
I asked Leo what are we using right now and left few more that seems for me could be useful.